app/views/invoices/show.html.erb

```diff
@@ -63,7 +63,7 @@
         </tbody>
       </table>
 
-      <% unless @invoice.paid? %>
+      <% unless !policy(Invoice).send_invoice? || @invoice.paid? %>
```

Comment:
What does this mean?
and why is the change needed?

Reply:
A user with the renting manager role can also access this page. However, they are not allowed to send an invoice, so for them the url should not appear on the page. So we check here with the policy for invoices (described in app/policies/invoice_policy.rb) is the person accessing the page is allowed to send an invoice

Reply:
Before this there was no renting manager role, so the check was unnecessary as only treasurers (who are allowed to send invoice) could access this page

Reply:
could this be rewrite so it is not a "unless not"statement to improve readabilty

Reply:
```suggestion
      <% if policy(Invoice).send_invoice? && !@invoice.paid? %>
```